qsimcirq/qsim_simulator.py

```diff
@@ -80,9 +80,13 @@ def __init__(self, qsim_options: dict = {},
             applied to all circuits run using this simulator. Accepted keys and
             their behavior are as follows:
                 - 'f': int (> 0). Maximum size of fused gates. Default: 2.
+                - 'r': int (> 0). Noisy repetitions (see below). Default: 1.
```

Comment:
To me passing the number of repetitions as a simulator option (these are simulator options not circuit options as stated a few lines above) doesn't make any sense. The number of repetitions is not a simulator option. It's an important parameter for the quantum trajectory method and that should be reflected in a higher level API. I think it should be passed as `repetitions` to `simulate_expectation_values_sweep` (or to a function with some better name). `simulate_expectation_values_sweep` with just one repetition doesn't really make a lot of sense for noisy simulations if the results get averaged in this function.

Reply:
This was suggested in my discussion with the Cirq team, with the primary reason being that setting "noisy repetitions" is analogous to setting simulation fidelity. Unless you're testing how different numbers of repetitions affect the quality of your result, you'll likely want a single value for this across multiple circuits.

The other motivating factor here is that `repetitions` doesn't make sense for most other `simulate_expectation_values` use cases (e.g. noiseless circuits or density matrix simulation). In the future, it might be worth the extra effort to have a separate method in Cirq (and we may revisit this then), but for the time being this is qsim-specific behavior.